multi-node/aws/pkg/config/config.go

```diff
@@ -74,6 +75,15 @@ func ClusterFromBytes(data []byte) (*Cluster, error) {
 	// as it will with RecordSets
 	c.HostedZone = WithTrailingDot(c.HostedZone)
 
+	if len(c.Subnets) == 0 {
+		c.Subnets = []Subnet{
```

Comment:
definitely appreciate the effort to keep backwards compatibility here. Given that this is clearly documented, we can later think about deprecating `instanceCIDR` and `availabilityZone`.